examples/WDC/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.3)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-wdc)
+
+hunter_add_package(WDC)
+find_package(WDC CONFIG REQUIRED)
+
+hunter_add_package(OpenSSL)
```

Comment:
If `WDC` depends on other packages then it should be hunterized:
* https://docs.hunter.sh/en/latest/creating-new/cmake-dependencies.html#maintenance